langserver/tracing.go

```diff
@@ -23,6 +23,12 @@ func (h *HandlerCommon) InitTracer(conn *jsonrpc2.Conn) {
 		return
 	}
 
+	if t := opentracing.GlobalTracer(); t != nil {
+		h.tracer = t
```

Comment:
go-langserver running not on sourcegraph.com. Like vscode/sublime it will likely provide useful to have.